ui/js/models/resultsets_store.js

```diff
@@ -885,6 +885,12 @@ treeherder.factory('ThResultSetStore', [
                         return;
                     }
 
+                    // there is already a polling task from loading the initial dataset
+                    // if the data has been loaded for appending.
+                    if (isAppend) {
```

Comment:
Unfortunately, this will cause the "next X" button to load more resultsets, but with no jobs.  So this approach won't have the desired effect.